### PR TITLE
sentry

### LINKED ISSUE #4903: Tracking errors in production
As far as I can see we don't have any means to track errors on production. Some users will report errors here, most won't but we can automate these reports. I think error tracking is necessary. What do you guys think?

I'd like to suggest [Rollbar](https://rollbar.com/). They have a free plan that we can use.

So from time to time we'd look in Rollbar and fix any error that occurs in production.

### PR DESCRIPTION
**Pull request by jywarren**
Fixes #4903

### CODE REVIEW

**Comment by siaw23** [comment]
Sweet. Good start! 

I'm wondering why you're bundling with Bundler 2 though 🤔

File: Gemfile.lock [hunk 1]
```diff
@@ -598,4 +599,4 @@ RUBY VERSION
    ruby 2.4.4p296
 
 BUNDLED WITH
-   1.17.3
+   2.0.1
```
Comment (jywarren) [inline Gemfile.lock]: i dunno i must've updated recently. is it a problem?

Comment (jywarren) [reply Gemfile.lock]: ugh, `You must use Bundler 2 or greater with this lockfile.`

Will try again downgrading bundler...

**Comment by siaw23** [comment]
Looks good to me. If they approve maybe you could put a link of where we can track the errors? Maybe in the README or wherever you find appropriate?

**Comment by jywarren** [comment]
Hmm, its coming through (can't invite yet, but will once we're approved) but there's a sprockets error: 

https://sentry.io/share/issue/9b54ec2922c0426683b689eea965ce87/ @siaw23 @icarito

**Comment by jywarren** [comment]
ok we are approved, inviting ppl now

**Comment by siaw23** [comment]
OK please send me and invite too. I'd love to look around. I think this will help us a lot in improving our application! Thanks for doing this Jeff!!

**Comment by icarito** [comment]
Friends I'm loving this! I really wanted to have a window into errors _as they were happening_!

**Pull request merged**